### PR TITLE
updated file basic-detailed-steps.js  #4475

### PR DESCRIPTION
**Pull request by khushishikhu**
i have tried to fix this issue. can you please review it .
#4475

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Nope. Not resolved

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -134,8 +134,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+       if($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')){
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [Missing space before opening brace.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 2]
```diff
@@ -134,8 +134,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
...
+        return false;
+      }
+      else{
+        return true;
+      }
     };
+
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:144]: Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 3]
```diff
@@ -134,8 +134,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+       if($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')){
+        return false;
+      }
+      else{
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:140]: Codacy found an issue: [Expected space(s) after "else".](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 1 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [Expected space(s) after "if".](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [Expected indentation of 6 spaces but found 7.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 4]
```diff
@@ -134,8 +134,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+       if($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')){
+        return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by khushishikhu** [comment]
@niranjan94  thank you so much sir for guiding me

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 5]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+       if ($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [Expected indentation of 6 spaces but found 7.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 6]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
...
+        return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 7]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+       if ( $('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [Expected indentation of 6 spaces but found 7.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 8]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
...
+        return false;
+       }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Expected indentation of 6 spaces but found 7.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 7 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 9]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ( $('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price') ) {
+        return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 10]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ( $('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price') ) {
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 11]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ( $('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
+        return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 12]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ( $('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:137]: Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 13]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
+        return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

Comment (khushishikhu) [reply app/components/forms/wizard/basic-details-step.js:139]: sir i am not getting this error. con you please help me out resolving this

**Comment by iamareebjamal** [comment]
They are automated reviews. Ignore them. Please focus on fixing the main issue and verify that it is resoved

**Comment by khushishikhu** [comment]
@iamareebjamal  sir i have tried to fix this out. please help me if there is something i am skiping.

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js [hunk 14]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
+         return false;
+      }
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 15]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
...
+      else {
+         return true;
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:141]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js [hunk 16]
```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
+         return false;
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:138]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by iamareebjamal** [comment]
You need to reproduce the problem and then verify that it doesn't exist anymore. The last time I checked it was still there

**Comment by khushishikhu** [comment]
@iamareebjamal  sir do i need to write my code this with ES6 classes? i tried to make it work out in basic-details-steps.js ......can you please help me what else i need to look up.

**Comment by iamareebjamal** [comment]
You can write it in ES6, E5 or ES3. What matters is that the bug is solved, which isn't

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js (hunk 16 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:138]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 14 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 15 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:141]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101&bid=18748634)

File: app/components/forms/wizard/basic-details-step.js (hunk 16 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:138]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 14 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:139]: Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

File: app/components/forms/wizard/basic-details-step.js (hunk 15 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:141]: Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)

**Comment by khushishikhu** [comment]
sir, is this issue is some what related to this #3534 . because i tried to take a reference from that but it still isn't working out. and is it needs to be fix from server also?

**Comment by iamareebjamal** [comment]
No, this is a frontend bug only

**Pull request closed without merging**